crates/pint-submit/src/main.rs

```diff
@@ -33,7 +36,10 @@ async fn run(args: Args) -> anyhow::Result<()> {
     let solution = serde_json::from_str::<Solution>(&from_file(solution).await?)?;
     let solution_ca = essential_hash::content_addr(&solution);
     print_submitting(&solution_ca);
-    let output = builder_client.submit_solution(&solution).await?;
+    let solutions = SolutionSet {
+        solutions: vec![solution],
+    };
+    let output = builder_client.submit_solution(&solutions).await?;
```

Comment:
Just noticing that this behaviour is a bit different to the previous behaviour. Now we only allow receiving an individual `Solution`, whereas previously we accepted a set of solutions. I can see it being useful to support both! For now, I'll change this to `--solution-set` and `SolutionSet`, to match the original behaviour more closely, but I'll open an issue to add support for submitting individual solutions (which might also help to simplify the "Deploy and Solve" chapter a little).

Reply:
Addressed in #144